### ISSUE TITLE
SQL assessment API / MaxDopFewNumaOver15 rule

### ISSUE THREAD

**Issue by rysalah** [microsoft/sql-server-samples#799]
Hi,

MaxDopFewNumaOver15 rule (Microsoft Ruleset 1.0.261) raises a warning when evaluated against a server with 32 logical cores on 2 NUMA nodes with a Maxdop equals to 16.

Message: "Update MAXDOP configuration option value: 16 – it should be half the number of logical processors per NUMA node"
This warning should be raised when a server has more than 16 cores by NUMA node.

sqlserver module 21.1.18226

Regards

https://docs.microsoft.com/en-us/sql/database-engine/configure-windows/configure-the-max-degree-of-parallelism-server-configuration-option?view=sql-server-ver15

![image](https://user-images.githubusercontent.com/44191918/89306985-da937700-d670-11ea-81bf-c024c0795e4a.png)

**Comment by ryakulevich** [comment]
Yes, this warning should not be raised at yours configuration. We will fix this issue in next release.
